src/app/utils/plug-descriptions.ts

```diff
@@ -265,28 +265,28 @@ function stripUsedStrings(
   communityInsight: Readonly<Perk>,
   usedStrings: ReadonlySet<string>
 ): Perk | undefined {
-  if (!communityInsight.simpleDescription) {
+  if (!communityInsight.description) {
     return;
   }
 
   // todo: only rebuild these arrays if they contain a duplicate line
 
-  const simpleDescription = communityInsight.simpleDescription.map((line) =>
-    line.lineText
-      ? {
-          ...line,
-          lineText: line.lineText.filter(
-            (content) => !content.text || !usedStrings.has(content.text)
-          ),
-        }
-      : line
-  );
-  if (!simpleDescription.some((line) => line.lineText?.length)) {
-    return;
-  }
+  // const description = communityInsight.description.map((line) =>
+  //   line.linesContent
+  //     ? {
+  //         ...line,
+  //         linesContent: line.linesContent.filter(
+  //           (content) => !content.text || !usedStrings.has(content.text)
+  //         ),
+  //       }
+  //     : line
+  // );
+  // if (!description.some((line) => line.linesContent?.length)) {
+  //   return;
+  // }
 
   return {
```

Comment:
I think we need to keep this code to account for when the community description is the same as the Bungie description e.g. **Invader Tracker**:

![dim-dupe-insight](https://user-images.githubusercontent.com/17512262/184622748-424febf6-1c30-4d79-8569-eae62b651320.png)

Reply:
Tbh I should rename that option its meant to be for all cases like this 
Clarity doesn't need this information either but because there are other applications using this data, not including DIM and clarity we sometimes add this information